README.md

```diff
@@ -20,6 +20,7 @@ dependencies {
     * [Making Cassandra more capable, faster, and more reliable](https://www.slideshare.net/scalar-inc/making-cassandra-more-capable-faster-and-more-reliable-at-apacheconhome-2020) at ApacheCon@Home 2020
     * [Scalar DB: A library that makes non-ACID databases ACID-compliant](https://www.slideshare.net/scalar-inc/scalar-db-a-library-that-makes-nonacid-databases-acidcompliant) at Database Lounge Tokyo #6 2020
     * [Transaction Management on Cassandra](https://www.slideshare.net/scalar-inc/transaction-management-on-cassandra) at Next Generation Cassandra Conference / ApacheCon NA 2019
+* [Scalar DB compatibility martix](docs/compatibility_matrix.md)
```

Comment:
```suggestion
* [Scalar DB supported databases](docs/compatibility_matrix.md)
```
Please move it under `Getting started`. Please also change the file name as well.

Reply:
Thank you for your suggestion.
Document position and file name also changed 2110824ef